__tests__/__snapshots__/UrqlClient_test.bs.js.snap

```diff
@@ -1,5 +1,26 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`UrqlClient Client with default requestPolicy should accept a default requestPolicy 1`] = `
+Client {
+  "activeOperations": Object {},
+  "createOperationContext": [Function],
+  "createRequestOperation": [Function],
+  "dispatchOperation": [Function],
+  "exchange": [Function],
+  "executeMutation": [Function],
+  "executeQuery": [Function],
+  "executeSubscription": [Function],
+  "fetch": undefined,
+  "fetchOptions": undefined,
+  "operations$": [Function],
+  "reexecuteOperation": [Function],
+  "requestPolicy": "cache-and-network",
```

Comment:
🎉 Just wanted to ensure that a different policy other than `cache-first` got picked up since, IIRC, that's already the default.